### PR TITLE
Update CI

### PR DESCRIPTION
**Pull request by IAlibay**
Mostly a test for if this will work with the current macos runner

Adds:
- [x] workflow trigger
- [x] cron trigger

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -46,29 +50,33 @@ jobs:
...
+
       - name: "Run tests"
         run: |
           pytest -n auto -v --cov=kartograf --cov-report=xml
+
       - name: "Test Notebooks"
+        if: ${{ matrix.os != 'macos' }}
         run: |
-          # openfe is a dep for the notebooks example, but not the package
-          micromamba install openfe=0.15
```
Comment (IAlibay) [inline .github/workflows/ci.yaml:67]: This was pinning to an old version of openfe & gufe.

File: .github/workflows/ci.yaml [hunk 2]
```diff
@@ -46,29 +50,33 @@ jobs:
...
         run: |
           micromamba info 
           micromamba list
+
       - name: "Run tests"
         run: |
           pytest -n auto -v --cov=kartograf --cov-report=xml
+
       - name: "Test Notebooks"
+        if: ${{ matrix.os != 'macos' }}
```
Comment (IAlibay) [inline .github/workflows/ci.yaml:71]: macos runners are super slow for nbval, there's probably no need to run notebook tests for them

File: .github/workflows/ci.yaml [hunk 3]
```diff
@@ -46,6 +50,8 @@ jobs:
           cache-downloads-key: downloads-${{ steps.date.outputs.date }}
           create-args: >-
             python=${{ matrix.python-version }}
+            lomap2
```
Comment (IAlibay) [inline .github/workflows/ci.yaml:53]: notebooks require lomap2 being installed

**Review by RiesBen — APPROVED** [review]
This looks good to me :)

**Pull request merged**